app/static/timer.js

```diff
@@ -0,0 +1,25 @@
+//credit for the countdowntimer: https://gist.github.com/Mak-Pro/0e1194d0f8696489a5c8bac72c8fa300
+fetch('/timer')
```

Comment:
Please add an event listener to trigger this script only when the page has fully loaded.

Reply:
Done